sql/core/src/test/scala/org/apache/spark/sql/streaming/TransformWithValueStateTTLSuite.scala

```diff
@@ -285,13 +285,13 @@ class TransformWithValueStateTTLSuite extends TransformWithStateTTLTest {
           .add("expiryTimestampMs", LongType, nullable = false)
         val schemaForValueRow: StructType = StructType(Array(StructField("__dummy__", NullType)))
         val schema0 = StateStoreColFamilySchema(
-          TimerStateUtils.getTimerStateVarName(TimeMode.ProcessingTime().toString),
-          schemaForKeyRow,
+          TimerStateUtils.getTimerStateVarNames(TimeMode.ProcessingTime().toString)._1, 0,
```

Comment:
Lets also add atleast a couple of tests for other composite types such as list, map etc ?